src/main/java/com/meilisearch/sdk/Documents.java

```diff
@@ -106,15 +106,16 @@ String getDocuments(String uid, int limit, int offset, List<String> attributesTo
      * @param uid Partial index identifier for the document
      * @param document String containing the document to add
      * @param primaryKey PrimaryKey of the document
-     * @return Meilisearch's Task API response
+     * @return Meilisearch's TaskInfo API response
      * @throws MeilisearchException if the client request causes an error
      */
-    Task addDocuments(String uid, String document, String primaryKey) throws MeilisearchException {
+    TaskInfo addDocuments(String uid, String document, String primaryKey)
+            throws MeilisearchException {
         String urlQuery = "/indexes/" + uid + "/documents";
```

Comment:
You can apply the URLBuilder here :)

Reply:
Yes, but I would like to create a specific PR to modify each method with the URLBuilder. WDYT?

Reply:
Yes this one will be changed during PR about documents changes